resources/rules/post_identifier_merge/ice_to_bio/miscrna_gene_type_assertions.clj

```diff
@@ -0,0 +1,22 @@
+;; ----------------------------------------------
+;; --------- miscRNA Gene Type Assertion ---------
+;; -----------------------------------------------
+`{:name "miscrna-gene-type-assertion"
+  :description "This rule asserts miscRNA gene type"
+  ;; miscRNA maps to SO: (so:)
+  :head ((?/egKabobDna rdfs/subClassOf obo/SO_XXXXXX))
```

Comment:
Looks like this rule was never completed. I think 'misc' means miscellaneous so we can probably us SO_0000356 'RNA' in place of SO_XXXXXX here.